app/controllers/internal/events_controller.rb

```diff
@@ -0,0 +1,140 @@
+module Internal
+  class EventsController < ::InternalController
+    before_action :load_buildings, only: %i[new edit create update]
+    layout "internal"
+
+    def index
+      @no_results = load_pending_events.blank?
+    end
+
+    def show
+      @event = GetIntoTeachingApiClient::TeachingEventsApi.new.get_teaching_event(params[:id])
+      @page_title = @event.name
+      if @event.status_id != GetIntoTeachingApiClient::Constants::EVENT_STATUS["Pending"]
```

Comment:
Do we need something similar to ensure pending events 404 on the main events#show path?